cps/main.c

```diff
@@ -66,6 +66,7 @@ cleanup_cps(void)
 	rte_kni_release(cps_conf.front_kni);
 	rte_timer_stop(&cps_conf.scan_timer);
 	destroy_mailbox(&cps_conf.mailbox);
+	destroy_mempool(cps_conf.mp);
 	rm_kni();
```

Comment:
Is `cleanup_cps()` destroing everything in reverse order?

Reply:
No, the current implementation does do it in the reverse order, I will change it.